docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+
+services:
+  postgresql:
+    build: ./
+    ports:
+      - "5433:5432"
+      - "6432:6432"
+    volumes:
+      - "lantern-suite-data:/var/lib/postgresql/data"
+      - "pg-socket-dir:/var/run/postgresql"
+    environment:
+      - "POSTGRES_HOST_AUTH_METHOD=trust"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+  http_server:
+    image: lanterndata/lantern-cli:latest-cpu
+    ports:
+      - "8080:8080"
+    volumes:
+      - "pg-socket-dir:/var/run/postgresql"
+    command: "start-server --host 0.0.0.0 --port 8080 --db-uri 'postgres://postgres@%2Fvar%2Frun%2Fpostgresql/postgres'"
```

Comment:
@var77 , are these `%2F` right?

Reply:
yes it is /var/run/postgresql just URL Encoded